tests/src/Interop/StringMarshalling/BSTR/BSTRTestNative.cpp

```diff
@@ -0,0 +1,195 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#include "platformdefines.h"
+
+WCHAR* strManaged = L("Managed\0String\0");
+size_t   lenstrManaged = 14; // the byte length of strManaged
```

Comment:
This is easy to compute at compile time: `ARRAYSIZE(strManaged) - 1` and the comment is wrong :-/ it is the string length. Please also update the `strManaged` type to be `WCHAR[]`.

Reply:
Below as well for `strNative`.